README.md

```diff
@@ -0,0 +1,151 @@
+The purpose of the corefx-testdata repo is to store corefx test file dependencies.
+
+Large test inputs (eg. over 1MB) should not bloat the corefx repo. Instead, they have their own dedicated repo, and the corefx build process pulls them via MyGet. The advantage of this is that their bloated history is not stored locally.
```

Comment:
Could you change instances of MyGet to dotnet blob feed? We just removed the `MyGet` consuption in corefx.

cc: @ViktorHofer

Reply:
My PR changes the README.md completely.

Reply:
@ViktorHofer why did you remove the instructions I uploaded? They were detailing step by step what do to and why. Your update to the readme file have no explanations for someone coming for the first time to participate in the project.